trade-finance-manager-ui/server/controllers/case/amendments/managersConditionsAndComments.controller.js

```diff
@@ -0,0 +1,135 @@
+const { format, fromUnixTime, getUnixTime } = require('date-fns');
+const api = require('../../../api');
+
+const { userCanEditManagersDecision } = require('../../helpers');
+const { AMENDMENT_STATUS } = require('../../../constants/amendments');
+const { UNDERWRITER_MANAGER_DECISIONS_TAGS } = require('../../../constants/decisions.constant');
+const { formattedNumber } = require('../../../helpers/number');
+
+const getManagersConditionsAndComments = async (req, res) => {
+  const { amendmentId, facilityId } = req.params;
```

Comment:
the `getAmendmentById` function (inside `api.js` file) is already wrapped inside a `try-catch` block. is it still required here?

Reply:
I still would as you are accessing `req` variable in case any of the property does not exists then it won't crash `tfm-ui` service.
Normally it wouldn't be required but due to some critical operations in your functions I would recommend it.